CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
You should declare `project` here, see:
- https://cgold.readthedocs.io/en/latest/tutorials/project.html#when-not-declared